api/WindowEventHandlers.json

```diff
@@ -467,7 +467,7 @@
                 "version_added": "57"
               },
               "ie": {
-                "version_added": null
+                "version_added": false
```

Comment:
Hah, this is bizarre, a "worker_support" entry on `WindowEventHandlers`. Sorting this out will be fun, but not in this PR I suggest since this is also about the "messageerror" event.